tests/src/Context/VariableContext.php

```diff
@@ -42,6 +42,22 @@ public function setVariable($name, $value) {
   }
 
   /**
+   * Deletes the value of a Drupal variable.
+   *
+   * The initial value of the value is remembered for later restore.
+   *
+   * @param string $name
+   *   Name of the variable.
+   */
+  public function deleteVariable($name) {
```

Comment:
The name of this function should be unsetVariable().
deleteVariable() is the opposite of addVariable().